content/dstu2/authorization.md

```diff
@@ -2,195 +2,1061 @@
 
-### Requesting an access token ###
-Once an authorization code has been acquired, a back-channel **POST** to the authorization server's token endpoint can be made to request an access token. Following the [OAuth 2.0][1] spec, the authorization server accepts a content type of **application/x-www-form-urlencoded** from your client application with the following information.
+The FHIR® Conformance resource can be retrieved by performing
+an HTTP GET against the resource located at the relative path
+of ``../Conformance`` from the FHIR<sup>®</sup> base URL.
 
-* grant_type - currently only support "authorization_code"
-* code
-* client_id
-* redirect_uri (only necessary if it was provided when the authorization code was requested)
+Within the structure of this document, two important URLs are
+advertised in an extension to ``Conformance.rest.security``, 
+identified via the URL of 
+``http://fhir-registry.smarthealthit.org/StructureDefinition/oauth-uris``.
 
-The authorization server authorize URL can be found in the security section of the [conformance document](../conformance).
+The first URL is the location of the authorization endpoint,
+which is further identified by the sub-extension URL "authorize".
 
-```
-grant_type=authorization_code&code={AUTHORIZATION_CODE}&client_id={YOUR_CLIENT_ID}&redirect_uri={YOUR CALLBACK URI, IF PROVIDED}
-```
+The second URL of importance is the location of the authorization
+server's token endpoint, which is further identified by the
+sub-extension URL "token".
 
-If successful, the authorization server will return a **200 OK** response with a content-type of [**application/json**][3] similar to the non-normative example below.
+The following is non-normative example of a conformance document
+containing this information:
 
-<pre class="body-response"><code class="language-javascript">{
-   "access_token":"eyJraWQiOiIyMDE2LTAyLTE2VDE2OjM2OjQ4LjY5MiIsInR5cCI6IkpXVCIsImFsZyI6IkVTMjU2In0.eyJzdWIiOiJIRDAxNjUxNCIsInVybjpjb206Y2VybmVyOmF1dGhvcml6YXRpb246Y2xhaW1zIjp7InZlciI6IjEuMCIsInRudCI6IjJjNDAwMDU0LTQyZDgtNGU3NC04N2I3LTgwYjViZDVmZGU5ZiIsImF6cyI6Im9ubGluZV9hY2Nlc3MifSwiYXpwIjoiZGV2anMiLCJpc3MiOiJodHRwczpcL1wvYXV0aG9yaXphdGlvbi5kZXZjZXJuZXIuY29tXC8iLCJleHAiOjE0NTU4OTkzOTEsImlhdCI6MTQ1NTg5ODc5MSwianRpIjoiOTQ1YjQ5ZjctMTFiMi00NmQ2LWEwZjctZGJmNjcxMzVmYTJlIiwidXJuOmNlcm5lcjphdXRob3JpemF0aW9uOmNsYWltczp2ZXJzaW9uOjEiOnsidmVyIjoiMS4wIiwicHJvZmlsZXMiOnsic21hcnQtdjEiOnsiYXpzIjoib25saW5lX2FjY2VzcyJ9fSwiY2xpZW50IjoiZGV2anMiLCJ1c2VyIjp7InN1YiI6IkhEMDE2NTE0IiwicGVyc29uYSI6InBhdGllbnQiLCJpZHNwIjoiOGUzM2FlM2QtNDg3Mi00ZDJmLWEwNTktNWYyMzA3ZmNiZjNiIiwiaWRzcFVyaSI6Imh0dHBzOlwvXC9hc3NvY2lhdGVzLmRldmNlcm5lci5jb21cL2FjY291bnRzXC9vcGVuaWQifSwidGVuYW50IjoiMmM0MDAwNTQtNDJkOC00ZTc0LTg3YjctODBiNWJkNWZkZTlmIn19.EnvJ8hRDIZcb9pKsfOaFGXhpAnfzi7rDMks0mAnAp0Lsbooe-JPoVPY8FT00xWP9JelbGdKGpoNLt0enGVGhnQ",
-   "token_type":"Bearer",
-   "expires_in":570,
-   "refresh_token":"1234-567890ab-cdef"
+<pre class="terminal">
```

Comment:
As-is this is missing one closing bracket and can you also update the formatting to
```
{
  "resourceType": "Conformance",
  "rest": [
    {
      "security": {
        "extension": [
          {
            "url": "http://fhir-registry.smarthealthit.org/StructureDefinition/oauth-uris",
            "extension": [
              {
                "url": "token",
                "valueUri": "https://example.org/token"
              },
              {
                "url": "authorize",
                "valueUri": "https://example.org/authorize"
              }
            ]
          }
        ]
      }
    }
  ]
}
```
?